### ISSUE TITLE
WCAG 1.4.3: Ensure the contrast between foreground and background colors meets WCAG 2 AA minimum contrast ratio thresholds (.maturityExperimental_g2AR.maturityBadge_GUVd)

### ISSUE THREAD

**Issue by WilliamBerryiii** [microsoft/hve-core#1918]
#### Issue
`Ensure the contrast between foreground and background colors meets WCAG 2 AA minimum contrast ratio thresholds` ([color-contrast](https://accessibilityinsights.io/info-examples/web/color-contrast))
#### Target application
[HVE Core](https://microsoft.github.io/hve-core/)
#### Element path
.collectionCard_PZk5:nth-child(11) > .collectionHeader_L9pK > .maturityExperimental_g2AR.maturityBadge_GUVd
#### Snippet
`<span class="maturityBadge_GUVd maturityExperimental_g2AR">Experimental</span>`
#### How to fix
    Fix any of the following:
      Element has insufficient color contrast of 2.25 (foreground color: #4cb3d4, background color: #eef9fd, font size: 8.3pt (11px), font weight: normal). Expected contrast ratio of 4.5:1
#### Environment
Microsoft Edge version 149.0.0.0
This accessibility issue was found using Accessibility Insights for Web 2.46.0 (axe-core 4.10.2), a tool that helps find and fix accessibility issues. Get more information & download this tool at [http://aka.ms/AccessibilityInsights](http://aka.ms/AccessibilityInsights).
